css/style.css

```diff
@@ -592,7 +645,7 @@ h4 {
 @media (min-width: 768px) {
   .header-top,
   .choose-screen {
-    display: none;
+    display: none !important;
```

Comment:
- [ ] As requested by the previous reviewer, Kindly according to the project requirement, 🤔  on the desktop size,  the closed button, and the menu mobile page must not be visible. Kindly correct this. You can use your `@media screen and (min-width: 768px) {`  to display none of your navigation mobile menu. 😃 

- Your design:

![Screenshot from 2022-10-11 02-09-57](https://user-images.githubusercontent.com/79058364/194970777-a715d5b6-2522-4a8e-853b-0b016c049fea.png)